src/Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj

```diff
@@ -16,11 +15,6 @@
     <RuntimeIdentifiers>win7-x64;win7-x86;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64;osx.10.11-x64;linux-x64</RuntimeIdentifiers>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System.Data.SqlClient" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="System.Data.SqlClient" Version="4.6.0" />
```

Comment:
Does the SqlClient package reference now come in transitively via SMO?

Reply:
no the <PackageReference> is in Directory.Build.props now